### PR TITLE
Updated Descriptions and Overview

### PR DESCRIPTION
**Pull request by cariadkeith**
Part 1 Draft for Updated OAS descriptions and Overview

### CODE REVIEW

File: resources/public/api/conf/1.0/Service Guide.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+# API Service Guide
+This initial discovery document shows developers and other users how to use UK Internal Market Scheme (UKIMS) API with their software.
+
+## API Overview 
+
+This API allows Fast Parcel Operators (FPOs) and 3rd party users to check if the holders of provided EORI Numbers have a valid UKIM authorisation.
+The aim is to avoid possible rejections of goods moving through from GB-NI, which could physically stop the B2B parcels.
+    
+The API is based on REST principles with a single GET method endpoint that returns data in JSON format. It uses standard HTTP error response codes. Use the API to request the UKIM Authorization Status of between 1 to 3000 EORIs passed as an array. You can use the API to send both small (up to 5MB in size) and large (up to 8MB in size) messages.
```
Comment (RobertLaverick) [inline resources/public/api/conf/1.0/Service Guide.md]: what is this about messages? That's not remotely true.

File: resources/public/api/conf/1.0/Service Guide.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    
+The API is based on REST principles with a single GET method endpoint that returns data in JSON format. It uses standard HTTP error response codes. Use the API to request the UKIM Authorization Status of between 1 to 3000 EORIs passed as an array. You can use the API to send both small (up to 5MB in size) and large (up to 8MB in size) messages. 
+    
+Note: The API endpoint relate only to Great Britain and Northern Ireland.
+
+
+Use the API to:
+
+Request the UKIM Authorization Status of 1-3000 EORIs passed as an array.
+You can use the API to send both small (up to 5MB in size) and large (up to 8MB in size) messages. ???
```
Comment (RobertLaverick) [inline resources/public/api/conf/1.0/Service Guide.md]: Again messages?

File: resources/public/api/conf/1.0/application.yaml [hunk 3]
```diff
@@ -1,33 +1,50 @@
...
+  title: UKIM EORI Authorisaton Checker
+  description: |
+    Overview
+    
+    This API allows Fast Parcel Operators (FPOs) and 3rd party users to check if the holders of provided EORI Numbers have a valid UKIM authorisation.
+    The aim is to avoid possible rejections of goods moving through from GB-NI, which could physically stop the B2B parcels.
+    
+    The API is based on REST principles with a single GET method endpoint that returns data in JSON format. It uses standard HTTP error response codes. 
+    
+    Use the API to request the UKIM Authorization Status of between 1 to 3000 EORIs passed as an array. You can use the API to send both small (up to 5MB in size) and large (up to 8MB in size) messages.
```
Comment (RobertLaverick) [inline resources/public/api/conf/1.0/application.yaml]: message sizes?

**Review by RobertLaverick — CHANGES_REQUESTED** [review]
1) can you make the commit names more meaningful? most of them are just. 'updated file' which isn't particularly meaning ful

2) You've got a whole mess of references to sending messages. We don't do that.

**Comment by cariadkeith** [comment]
Removed wrong references to message sizes

**Comment by cariadkeith** [comment]
Latest updates and fixes

File: resources/public/api/conf/1.0/application.yaml [hunk 4]
```diff
@@ -1,34 +1,49 @@
 openapi: 3.0.3
 info:
-  title: UKIM Auth Checker
-  description: |-
-    This API allows invited users to validate if the holders of EORI Numbers have a UKIM authorisation.
-    
-    Some useful links:
-    - [Github]()
-    - [Source API definition]()
+  title: UKIM EORI Authorisaton Checker UKIM EORI Authorisaton Checker Reference Guide
```
Comment (RobertLaverick) [inline resources/public/api/conf/1.0/application.yaml]: Repetition seems confusing

File: resources/public/api/conf/1.0/application.yaml [hunk 5]
```diff
@@ -1,34 +1,49 @@
...
+        - Authorisations
+      summary: |
+        Request Authorisation Status for provided EORIs
+      description: |
+        Using this method you can request the UKIM Authorization Status of between 1 and 3000 EORIs. EORIs are taken as a Comma Separated List (CSV) and stored as a string type array . 
+        There are two string query header parameters - eori and date.  
       operationId: requestAuth
       parameters:
-        - name: eoris
+        - name: eori
```
Comment (RobertLaverick) [inline resources/public/api/conf/1.0/application.yaml]: it is eoris, not eori

File: resources/public/api/conf/1.0/application.yaml [hunk 6]
```diff
@@ -37,39 +52,49 @@ paths:
...
+        id: l4z9id8uvcvj0
 components:
   schemas:
     Authorisations:
       type: object
       properties:
         date:
           type: string
-          example: 2024-02-31
+          example: '2024-02-31'
```
Comment (RobertLaverick) [inline resources/public/api/conf/1.0/application.yaml]: Do we need the single quotes? I'm not sure how this renders but they wouldn't need the quotes in the query string

**Review by RobertLaverick — CHANGES_REQUESTED** [review]

File: resources/public/api/conf/1.0/application.yaml (hunk 5 above)
Comment (cariadkeith) [reply resources/public/api/conf/1.0/application.yaml]: Fixed this

File: resources/public/api/conf/1.0/application.yaml (hunk 4 above)
Comment (cariadkeith) [reply resources/public/api/conf/1.0/application.yaml:3]: Not sure what you mean here Rob

File: resources/public/api/conf/1.0/application.yaml (hunk 6 above)
Comment (cariadkeith) [reply resources/public/api/conf/1.0/application.yaml]: removed single quotes

File: resources/public/api/conf/1.0/application.yaml (hunk 4 above)
Comment (cariadkeith) [reply resources/public/api/conf/1.0/application.yaml]: Ah sorry fixed that

**Review by RobertLaverick — APPROVED** [review]

**Comment by cariadkeith** [comment]
First merge hopefully to QA

**Pull request merged**